### ISSUE TITLE
CC | CI: SEV build failing as suffix missing from initrd.img

### ISSUE THREAD

**Issue by stevenhorsman** [kata-containers/kata-containers#5938]
When running the SEV tests we hit the following:
```
INFO: kill stale process
09:27:21 /opt/confidential-containers/share/defaults/kata-containers/configuration.toml: file /opt/confidential-containers/share/kata-containers/kata-containers-initrd.img does not exist
09:27:21 /opt/confidential-containers/share/defaults/kata-containers/configuration.toml: file /opt/confidential-containers/share/kata-containers/kata-containers-initrd.img does not exist
```

It doesn't happen on every test run though e.g. http://jenkins.katacontainers.io/view/CCv0/job/tests-CCv0-ubuntu-20.04_sev-x86_64-CC_SEV_CRI_CONTAINERD_K8S-PR/195/ passed

### Explanation

I think the problem was introduced in https://github.com/kata-containers/kata-containers/pull/5835/files with the change to add the sev suffix for the initrd for caching with https://github.com/kata-containers/kata-containers/blob/70b617db98886a67754b5b477909f57f7ac416ce/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L283

From the jenkins logs we can see that it is building `./opt/confidential-containers/share/kata-containers/kata-containers-initrd-sev.img` but the sev config.toml was still pointing at the old name:

```
[hypervisor.qemu]
path = "/opt/confidential-containers/bin/qemu-system-x86_64"
kernel = "/opt/confidential-containers/share/kata-containers/vmlinuz-sev.container"
initrd = "/opt/confidential-containers/share/kata-containers/kata-containers-initrd.img"
```

So I'll create a fix to update the sev config to point at the new path and try and resolve this.

The next question becomes - why is it still working on some jobs and why didn't we spot the failure on the PR that introduced it. I have a theory for this - Looking at the two different sev node being used. Job that run on `amd-coco-ci-ubuntu2004-001` (and don't fail with the frequent network problems) pass, and jobs running on `amd-milan-coco-ci-ubuntu2004-001` fail. On Friday Ryan updated the disk space of that node to add 3T, so I'm guessing at the same time some of the old files were deleted.

I think that `amd-coco-ci-ubuntu2004-001` still has an old copy of the initrd in `/opt/confidential-containers/share/kata-containers/kata-containers-initrd.img`, so it still passes, whereas this was cleared out from the milan node, so the tests fail.

It means the passing tests may be false positives and they aren't running the latest build code and indicated a wider problem that (for AMD SEV at least) old artifacts are hanging around and could lead to incorrect test results, so I think that needs resolving to stop this happening again (and being a massive pain to debug)

**Comment by fitzthum** [comment]
Hm yeah #5835 should never have passed the SEV tests most likely.

**Comment by stevenhorsman** [comment]
Closed with #5939 merging.
